domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -350,6 +354,21 @@ static String createPassword(String password) throws Exception {
     return password;
   }
 
+  /**
+   * This instance's PgPool for database connections.
+   *
+   * Take care to execute "SET ROLE ..." when {@link #sharedPgPool} is true; consider using
+   * one of the with... methods that automatically execute "SET ROLE ...".
+   *
+   * @see #withConnection(Function)
+   * @see #withConn(Function)
+   * @see #withTrans(Function)
+   * @see #withTransaction(Function)
+   */
+  PgPool getReaderClient() {
+    return clientReader;
+  }
```

Comment:
There should also be a setReaderClient()

Reply:
done